### ISSUE TITLE
Creating Vue App fails with Alpha 20

### ISSUE THREAD

**Issue by jankapunkt** [meteor/meteor#12937]
## Summary

Creating a new Vue app fails with package version conflicts in 3.0 Alpha 20.

## Reproduction:

```shell
$ meteor create --vue --release=3.0-alpha.20 some-vue-app
```

Yields in the following error:

```shell
Using vue skeleton                                                                 
Something has happened while creating your app using git clone.
Will use cached version of skeletons.
Error message:  Using release option
=> Errors while creating your project         
                                              
While selecting package versions:
error: Conflict: Constraint webapp@1.13.1 is not satisfied by webapp 2.0.0-alpha300.20.
Constraints on package "webapp":
* webapp@~2.0.0-alpha300.20 <- top level
* webapp@2.0.0-alpha300.20 <- meteor-base 1.5.2-alpha300.20
* webapp@2.0.0-alpha300.20 <- routepolicy 1.1.2-alpha300.20 <- ddp-server 3.0.0-alpha300.20 <- ddp 1.4.2-alpha300.20 <-
allow-deny 2.0.0-alpha300.20 <- mongo 2.0.0-alpha300.20
* webapp@2.0.0-alpha300.20 <- routepolicy 1.1.2-alpha300.20 <- ddp-server 3.0.0-alpha300.20 <- ddp 1.4.2-alpha300.20 <-
meteor-base 1.5.2-alpha300.20
* webapp@2.0.0-alpha300.20 <- routepolicy 1.1.2-alpha300.20 <- webapp 2.0.0-alpha300.20 <- jorgenvatle:vite-bundler
1.9.1
* webapp@2.0.0-alpha300.20 <- ddp-server 3.0.0-alpha300.20 <- ddp 1.4.2-alpha300.20 <- allow-deny 2.0.0-alpha300.20 <-
mongo 2.0.0-alpha300.20
* webapp@2.0.0-alpha300.20 <- ddp-server 3.0.0-alpha300.20 <- ddp 1.4.2-alpha300.20 <- meteor-base 1.5.2-alpha300.20
* webapp@2.0.0-alpha300.20 <- autoupdate 2.0.0-alpha300.20 <- hot-code-push 1.0.5-alpha300.20 <- hot-module-replacement
0.5.4-alpha300.20
* webapp@2.0.0-alpha300.20 <- mongo 2.0.0-alpha300.20
* webapp@1.13.1 <- jorgenvatle:vite-bundler 1.9.1

Conflict: Constraint caching-compiler@1.2.2 is not satisfied by caching-compiler 2.0.0-alpha300.20.
Constraints on package "caching-compiler":
* caching-compiler@~2.0.0-alpha300.20 <- top level
* caching-compiler@2.0.0-alpha300.17 <- caching-html-compiler 2.0.0-alpha300.17 <- static-html 1.3.3-alpha300.20
* caching-compiler@1.2.2 <- jorgenvatle:vite-bundler 1.9.1
```

**Comment by JorgenVatle** [comment]
Hey there. 👋

The beta release `jorgenvatle:vite-bundler@2.0.0-beta.8` for the Vite bundler is compatible with Meteor v3. The latest stable release also resolves the version mismatch there, but it seems like Fibers gets added as a dependency whenever it's published using the Meteor v2 CLI tool.

**Comment by jankapunkt** [comment]
Thank you @JorgenVatle coming back at this with RC.4 I still can't run a new Vue/Vite project out of the box.

Repro:

```shell
$ meteor create --vue --release=3.0-rc.4 some-vue-app
$ meteor npm install --save-dev meteor-vite@1.8.0 # skeleton has not the latest
$ meteor
```

Then navigating to `localhost:3000` and without doing any further config I immediately get the following error:

```
ERROR  Could not extract name from package in: .meteor/local/build/programs/web.browser/packages/meteor.js
undefined

  // File content for: .meteor/local/build/programs/web.browser/packages/meteor.js 

...  very long output of the meteor.js bundled file'c content

    at parseMeteorPackage (file:///home/username/dev/videos/hypertravel/node_modules/meteor-vite/dist/bin/worker.mjs:179:11)
    at async _MeteorPackage.parse (file:///home/username/dev/videos/hypertravel/node_modules/meteor-vite/dist/bin/worker.mjs:1001:35)
    at async Context.load (file:///home/username/dev/videos/hypertravel/node_modules/meteor-vite/dist/bin/worker.mjs:1476:29)
    at async Object.load (file:///home/username/dev/videos/hypertravel/node_modules/vite/dist/node/chunks/dep-372dab03.js:40584:32)
    at async loadAndTransform (file:///home/username/dev/videos/hypertravel/node_modules/vite/dist/node/chunks/dep-372dab03.js:36936:24

Click outside or fix the code to dismiss.
You can also disable this overlay by setting server.hmr.overlay to false in vite.config.js. 
``` 


Note: The `jorgenvatle;vite-bundler` is at `2.0.0` the newest major release from June 24.

**Comment by JorgenVatle** [comment]
Hey there.

Haven't tried that Meteor release yet. But I suspect there's a chance the 2.0.0 release might've been improperly published. I'll try to do some testing over the weekend to verify. In the meantime, could you try with the latest beta release and see if that makes a difference? `jorgenvatle:vite-bundler@2.0.0-beta.14`

**Comment by jankapunkt** [comment]
Will check that out, thanks 👍

**Comment by JorgenVatle** [comment]
Just tried out Meteor v3.0-rc.4 with `jorgenvatle:vite-bundler@2.0.0` things seem to be running fine on my end. Perhaps the `meteor-vite` npm package is out of date?

**Comment by jankapunkt** [comment]
Sorry, I did not realize there is a newer version available. The command line message told me to install 1.8.0. I will check this out to see if it works with the latest. Thank you

**Comment by JorgenVatle** [comment]
Ahah, that's all my fault. The invalid version notice appeared with the first beta release but I kept forgetting to fix it with every release since. 🙈

Sorry to have let it take up this much of your time.

**Comment by jankapunkt** [comment]
No worries, I tried with

```shell
$ meteor npm install --save-dev meteor-vite@latest
```

and it worked out to be fine

**Comment by nachocodoner** [comment]
Closing this issue since it seems solved, Vue 3 works on Meteor 3.

``` bash
meteor npm i -D vite@4
meteor npm i meteor-vite@latest
meteor add jorgenvatle:vite-bundler@2.0.0 --allow-incompatible-update
```

**Comment by mattsouth** [comment]
When I just tried to create a vue skeleton app (with Meteor 3.0.3 on ubuntu 22.0) I saw the constraint conflict.  Is this expected?  

```
me@mymachine:~/Software$ meteor create --vue meteor-vue-3.0.3
Using vue skeleton                                                                 
=> Errors while creating your project         
                                              
While selecting package versions:
error: Conflict: Constraint webapp@1.13.1 is not satisfied by webapp 2.0.2.
Constraints on package "webapp":
* webapp@~2.0.1 <- top level
* webapp@2.0.0 <- meteor-base 1.5.2
* webapp@2.0.0 <- routepolicy 1.1.2 <- ddp-server 3.0.1 <- ddp 1.4.2 <- allow-deny 2.0.0 <- mongo 2.0.2
* webapp@2.0.0 <- routepolicy 1.1.2 <- ddp-server 3.0.1 <- ddp 1.4.2 <- meteor-base 1.5.2
* webapp@2.0.0 <- routepolicy 1.1.2 <- webapp 2.0.2 <- meteor-base 1.5.2
* webapp@2.0.1 <- ddp-server 3.0.1 <- ddp 1.4.2 <- allow-deny 2.0.0 <- mongo 2.0.2
* webapp@2.0.1 <- ddp-server 3.0.1 <- ddp 1.4.2 <- meteor-base 1.5.2
* webapp@2.0.0 <- autoupdate 2.0.0 <- hot-code-push 1.0.5 <- hot-module-replacement 0.5.4
* webapp@2.0.1 <- mongo 2.0.2
* webapp@1.13.1 <- vite:bundler 0.1.9

Conflict: Constraint caching-compiler@1.2.2 is not satisfied by caching-compiler 2.0.0.
Constraints on package "caching-compiler":
* caching-compiler@~2.0.0 <- top level
* caching-compiler@2.0.0 <- caching-html-compiler 2.0.0 <- static-html 1.3.3
* caching-compiler@1.2.2 <- vite:bundler 0.1.9

Something has happened while creating your app using git clone.
Will use cached version of skeletons.
Error message:  undefined
Created a new Meteor app in 'meteor-vue-3.0.3'.                                    

To run your new app:                          
  cd meteor-vue-3.0.3                         
  meteor                                      
                                              
If you are new to Meteor, try some of the learning resources here:
  https://www.meteor.com/tutorials            
                                              
When you’re ready to deploy and host your new Meteor application, check out Cloud:
  https://www.meteor.com/cloud                
```

**Comment by jankapunkt** [comment]
I think the `vite:bundler` is the one that is to be replaced by the npm package, is that correct @nachocodoner ?

**Comment by mattsouth** [comment]
I can see this issue remains in 3.0.4 but is resolved in 3.1-rc.0.
